utils/records-decorator.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+function decorateForSearch(records, fieldsSearched, searchValue) {
+  if (records[0] && records[0].dataValues) {
+    records = records.map((record) => record.get({ plain: true }));
+  }
```

Comment:
You can get rid of those 3 lines and do `record.attributes` to simplify your code